### ISSUE TITLE
New features to add: PolicyGradient, OpenAI Gym, Keras, Asynchronous Training

### ISSUE THREAD

**Issue by lsqshr** [nivwusquorum/tensorflow-deepq#33]
Since I am using this repo in my work as well, some of the features I found might be useful:
1. Continuous control: A couple of recent papers have been published to show that deep NN would enable complex high dimensional controls with policy gradient. It seems the `continuous` branch was initiated but has been inactive for a while (is it totally abandoned?). 
2. Support OpenAI gym to make continuous control easier and bench-marked
3. Support Keras to shrink the code base regarding to tensorflow, also make it easier to program more complex networks 
4. Asynchronous training with multi-agent

The latter three are supported in [Asyncronous RL in Tensorflow + Keras + OpenAI's Gym](https://github.com/coreylynch/async-rl) However I do prefer the architecture here to keep things well structured.
